apps/ledger-live-desktop/src/renderer/hooks/useDeeplinking.ts

```diff
@@ -310,6 +310,10 @@ export function useDeepLinkHandler() {
           if (path?.startsWith("protect")) {
             navigate(`/recover/${path}`, undefined, search);
           } else {
+            // load discover section wtih an initial category filter parameter, e.g. /discover?category=staking
+            if (!path && query.category) {
+              navigate(`/platform`, query);
+            }
```

Comment:
Same here, I'm not confident that we actually need this as query is already passed below even if we don't have an `appId` (named `path` here)

Reply:
Ah yes the `/` is still a match with the original Route even if there's only a search param after it.
Do you see any issues with passing the `search` (if there) to all platform navigations including those to specific app ids?
i.e. 
```
            navigate(`/platform/${path ?? ""}`, query, search);
```